Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-UserDeletedFromDatabase.yaml

```diff
@@ -1,6 +1,9 @@
 id: 7b8fa5f5-4f5b-4698-a4cf-720bbb215bea
 name: SQL User deleted from Database
 description: |
+    This hunting query identifies deletion of user from SQL Database
```

Comment:
"This hunting query identifies deletion of user from SQL Database. It relies on the SQLEvent KQL Parser function."